### PR TITLE
Removed the min sliders from the optimiser

### PR DESCRIPTION
**Pull request by ryan-rushton**

### CODE REVIEW

File: src/app/loadout-builder/processWorker/process.ts [hunk 1]
```diff
@@ -65,7 +65,9 @@ function insertIntoSetTracker(
             armorSetIndex < currentStatMix.armorSets.length;
             armorSetIndex++
           ) {
-            if (armorSet.maxPower > currentStatMix.armorSets[armorSetIndex].maxPower) {
+            if (
+              getPower(armorSet.armor) > getPower(currentStatMix.armorSets[armorSetIndex].armor)
```
Comment (ryan-rushton) [inline src/app/loadout-builder/processWorker/process.ts:69]: I pulled the maxPower attribute out of the loadouts so we have slightly less to calculate in the hot part of the process loop. This condition should rarely be triggered and we calculate it again when rendering.

File: src/app/loadout-builder/generated-sets/GeneratedSet.tsx [hunk 2]
```diff
@@ -72,14 +73,16 @@ function GeneratedSet({
     set.armor.every((items) => loadout.items.map((item) => item.id).includes(items[0].id))
   );
 
+  const items = set.armor.map((items) => items[0]);
```
Comment (bhollis) [inline src/app/loadout-builder/generated-sets/GeneratedSet.tsx:76]: Might be worth seeing if this component gets re-rendered a lot, and if so, wrapping this and the `getPower` in a useMemo. If it only really re-renders when those would be different anyway, don't bother.

**Review by bhollis — APPROVED** [review]
Looks good. Let's merge after the prod release in case somebody relied on this for their Workflow™️

File: src/app/loadout-builder/generated-sets/GeneratedSet.tsx (hunk 2 above)
Comment (ryan-rushton) [reply src/app/loadout-builder/generated-sets/GeneratedSet.tsx:76]: K will do

**Pull request merged**